medusa/__main__.py

```diff
@@ -2070,6 +2078,51 @@ def shutdown(self, event):
             logging.shutdown()
             os._exit(0)  # TODO: Remove in another PR. There's no need for this one.
 
+    @staticmethod
+    def configure_unrar():
```

Comment:
I think you should start at the very beginning, maybe something like [this](https://www.amazon.com/Python-Best-Practices-Programming-Code-ebook/dp/B01N4LDZT1) could be a good start and it's almost free. Otherwise try to google about good coding practices in general (not necessarily about Python). How to write simple, clean functions, how to write good comments, readable code, etc.